Assets/MRTK/SDK/Experimental/Joystick/JoystickController.cs

```diff
@@ -0,0 +1,162 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Joystick
+{
+    /// <summary>
+    /// Example script to demonstrate joystick control in sample scene
+    /// </summary>
+    public class JoystickController : MonoBehaviour
+    {
+        public GameObject ObjectToManipulate;
```

Comment:
We recommend that [all fields be private with public properties for access (if needed)](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#encapsulation).

You can use `[SerializeField]` on private fields if you need them to show up in the inspector.

Reply:
This will likely also require some field naming updates: https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#naming-conventions